src/commands/moderation/unban.ts

```diff
@@ -7,7 +7,7 @@ type UserLike = User | Snowflake
 
 export const unban: CommandDefinition = {
     name: 'unban',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?